packages/workbox-cache-expiration/CacheExpirationManager.mjs

```diff
@@ -88,46 +91,52 @@ class CacheExpirationManager {
 
     // Use a Set to remove any duplicates following the concatenation, then
     // convert back into an array.
-    const urls = [...new Set(oldEntries.concat(extraEntries))];
+    const allUrls = [...new Set(oldEntries.concat(extraEntries))];
     await Promise.all([
-      this._deleteFromCache(urls),
-      this._deleteFromIDB(urls),
+      this._deleteFromCache(allUrls),
+      this._deleteFromIDB(allUrls),
     ]);
 
     if (process.env.NODE_ENV !== 'production') {
+      // TODO break apart entries deleted due to expiration vs size restraints
       _private.logger.groupCollapsed(
-        `Expired ${urls.length}entries have been removed from the cache.`);
+        `Expired ${allUrls.length}entries have been removed from the cache.`);
```

Comment:
Nit: This log message formatting looks a bit weird. Can you confirm the spacing?

Reply:
Argh - that needs a space. will add in.